frontend/src/components/widgets/utils.test.js

```diff
@@ -0,0 +1,54 @@
+import { getAgeString } from "./utils";
+
+test("getAgeString function doesn't break if null date", () => {
+  expect(getAgeString(null)).toBe("");
+  expect(getAgeString(undefined)).toBe("");
+});
+
+test("getAgeString function returns correct string when age is 1 month", () => {
+  const date = new Date();
+  const card = {
```

Comment:
While we are at it, let's lclean these tests up the rest of the way. `getAgeString` doesn't need to know about cards. It doesn't need to know anything except a time.

This is much cleaner:
```
date = date.setDate(today.getDate() - 31)
expect(getAgeString(date)).toBe("1 month ago");
```